tests/Oryx.Integration.Tests/Database/DatabaseTestsBase.cs

```diff
@@ -81,6 +84,24 @@ protected DatabaseTestsBase(ITestOutputHelper outputHelper, Fixtures.DbContainer
                 });
         }
 
+        private List<EnvironmentVariable> GetEnvironmentVariableList()
```

Comment:
We have 2 different methods called `GetEnvironmentVariableList` that are returning a list of environment variables with essentially the same logic. Can we convert these into a single method that extends an env var list that performs the required logic?

E.g. something along the lines of

```csharp
public List<EnvironmentVariable> AddStorageAccountEnvironmentVariables(this List<EnvironmentVariable> envVarList) 
{
  var testStorageAccountUrl = Environment.GetEnvironmentVariable(SdkStorageConstants.TestingSdkStorageUrlKeyName);
  var sdkStorageUrl = string.IsNullOrEmpty(testStorageAccountUrl) ? SdkStorageConstants.PrivateStagingSdkStorageBaseUrl : testStorageAccountUrl;

  envVarList.Add(new EnvironmentVariable(SdkStorageConstants.SdkStorageBaseUrlKeyName, sdkStorageUrl));

  if (sdkStorageUrl == SdkStorageConstants.PrivateStagingSdkStorageBaseUrl)
  {
      string stagingStorageSasToken = Environment.GetEnvironmentVariable(SdkStorageConstants.PrivateStagingStorageSasTokenKey) ??
      this.GetKeyVaultSecretValue(SdkStorageConstants.OryxKeyvaultUri, SdkStorageConstants.StagingStorageSasTokenKeyvaultSecretName);
      envVarList.Add(new EnvironmentVariable(SdkStorageConstants.PrivateStagingStorageSasTokenKey, stagingStorageSasToken));
  }

  return envVarList;
}
```

so then we could do the following above:

`_dbFixture.GetCredentialsAsEnvVars().AddStorageAccountEnvironmentVariables()`

and the following later on:

`SqlServerDbTestHelper.GetEnvironmentVariables().AddStorageAccountEnvironmentVariables()`

Reply:
a common work item which will address the issues: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1848573